src/cheat_gui.cpp

```diff
@@ -112,13 +113,14 @@ static int32 ClickChangeDateCheat(int32 new_value, int32 change_direction)
 	TimerGameCalendar::Date new_date = TimerGameCalendar::ConvertYMDToDate(new_value, ymd.month, ymd.day);
 
 	/* Shift cached dates before we change the date. */
-	for (auto v : Vehicle::Iterate()) v->ShiftDates(new_date - TimerGameCalendar::date);
-	LinkGraphSchedule::instance.ShiftDates(new_date - TimerGameCalendar::date);
+	for (auto v : Vehicle::Iterate()) v->ShiftDates(new_date - TimerGameEconomy::date);
+	LinkGraphSchedule::instance.ShiftDates(new_date - TimerGameEconomy::date);
```

Comment:
This is now a bit weird; I know it will be fixed in later PRs, but you calculate `new_date` based on `Calendar`, then use `Economy` to do the delta. And yes, of course they are both set to the same value later, but it just reads a bit weird.

Not sure it needs fixing, just observing.